package.json

```diff
@@ -109,13 +110,13 @@
     "marked": "^0.6.0",
     "prettier": "^1.14.3",
     "rimraf": "2.6.3",
-    "rollup": "1.15.2",
+    "rollup": "git://github.com/jstarry/rollup.git#fix-resolved-id-result-bug",
```

Comment:
I'd prefer to wait for your upstream fix to be merged and released rather than merging this PR with this github dependency.  This isn't a super urgent issue though so we can wait to see what happens upstream

Reply:
@mvines Yeah great call! I added the last commit with the message "Check that rollup fix works" ;) I was planning to remove it once I saw if CI passed